signing_service/signing_service/tests/tests.py

```diff
@@ -107,14 +118,68 @@ def test_that_signing_service_should_reraise_different_socket_erros(self):
         mock_socket_close.assert_called_once()
 
 
+class SigningServiceHandleConnectionTestCase(TestCase):
+
+    def setUp(self):
+        self.host = '127.0.0.1'
+        self.port = 8000
+        self.initial_reconnect_delay = 2
+        # TODO: Use predefined ones when SigningService will accept Concent private key as parameter.
+        self.concent_public_key = b'\xf3\x97\x19\xcdX\xda\x86tiP\x1c&\xd39M\x9e\xa4\xddb\x89\xb5,]O\xd5cR\x84\xb85\xed\xc9\xa17e,\xb2s\xeb\n1\xcaN.l\xba\xc3\xb7\xc2\xba\xff\xabN\xde\xb3\x0b\xa6l\xbf6o\x81\xe0;'
```

Comment:
Please use `CONCENT_PUBLIC_KEY` which you generate in the top of file

Reply:
Please read the comment above this line ;)

Reply:
I have updated issue description yesterday. Please add command-line options for this in this pull request. And please remove all hard-coded keys from the code. We rarely ever want to have keys hard-coded so if you see no other choice, it's very likely that it's my mistake.

Reply:
:+1: done